modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -467,7 +476,7 @@ public void initJail(final String js, final Callback callback) {
         Thread thread = new Thread() {
             @Override
             public void run() {
-                Statusgo.InitJail(js);
+                jail.initJail(js);
```

Comment:
Does this respect JSC_ENABLED?

Reply:
Seems like yes `public class OttoJail implements Jail {`

Reply:
Yes it does https://github.com/status-im/status-react/pull/2450/files#diff-b35d072cfa0e68957f448ee1ae0de901R52